Mathlib/CategoryTheory/MorphismProperty/Limits.lean

```diff
@@ -605,6 +642,35 @@ lemma isStableUnderCoproductsOfShape_of_isStableUnderFiniteCoproducts
     W.IsStableUnderCoproductsOfShape J :=
   IsStableUnderFiniteCoproducts.isStableUnderCoproductsOfShape J
 
+/-- The condition that a property of morphisms is stable by coproducts. -/
+class IsStableUnderCoproducts : Prop where
```

Comment:
Should this be `IsStableUnderCoproductsOfSize` in analogy with the naming in the limits library? And should it be `@[pp_with_univ]`?

Reply:
Actually, we have `HasCoproducts`, but I will add `@[pp_with_univ]`.